### PR TITLE
Add in error handling to locale

### PR DESCRIPTION
**Pull request by lamtu1**
Add in a way to handle the param for locale and not raise the `I18n::InvalidLocale`.

### CODE REVIEW

**Comment by lamtu1** [comment]
`DEMO:` Showcasing the error handling of locale.

https://github.com/osulp/Scholars-Archive/assets/111306182/6df46475-f1b0-441d-82df-d044a7b4471f

**Comment by lamtu1** [comment]
fixes #1880

**Comment by lamtu1** [comment]
Ready for review/approve and merge for QA.

File: app/controllers/concerns/hyrax/controller.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+# OVERRIDE: Setup an override on locale to render header
+module Hyrax::Controller
```
Comment (CGillen) [inline app/controllers/concerns/hyrax/controller.rb:4]: See that this is a module & not a class. We should be able to find where this is imported and override the method there without pulling in this whole module

Comment (CGillen) [reply app/controllers/concerns/hyrax/controller.rb:4]: https://github.com/osulp/Scholars-Archive/blob/main/app/controllers/application_controller.rb#L12

**Review by CGillen — CHANGES_REQUESTED** [review]

**Review by CGillen — APPROVED** [review]

**Pull request merged**